solr/core/src/test/org/apache/solr/request/macro/TestMacroExpander.java

```diff
@@ -37,8 +37,7 @@ public void testExamples() {
       me = new MacroExpander(testParams, failOnMissingParams);
     }
 
-    // default examples: https://cwiki.apache.org/confluence/display/solr/Parameter+Substitution
```

Comment:
removed the link to https://cwiki.apache.org/confluence/display/solr/Parameter+Substitution because of 404 Not Found